battleship.py

```diff
@@ -43,7 +44,8 @@ def makeModel(data):
 Returns: None
 '''
 def makeView(data, userCanvas, compCanvas):
-
+    userCanvas=drawGrid(data,userCanvas,data["user board"],True) 

```

Comment:
update the separation between variables with _ or uppercase character

Reply:
changed mam